backend/server.js

```diff
@@ -0,0 +1,23 @@
+// var models = require('./models');
+// var config = require('./config');
+// var express = require('express');
+// var app = express();
+import { version } from '../package.json';
+import express from 'express';
+const debug = require('debug')('moviebox');
+
+const api = express.Router(); // eslint-disable-line new-cap
```

Comment:
This is the way to disable `eslint` on specific lines of code.
In this situation, `Router` starts with capital letter and `eslint` expects `new` to be used with capital letters. If we don't, then it's considered a linting error.

Disabling the `eslint` rule on the specific line of code fixes the issue.